swag/blockchain/synced_blockchain.py

```diff
@@ -44,11 +42,11 @@ async def append(self, block):
         if isinstance(block, YfuGenerationBlock):
             avatar_mes = await self._channel.send(
```

Comment:
In `avatar_mes`, what is `mes`?

Reply:
Ah, message

Reply:
Maybe the full name would be clearer here

Reply:
Resolved in b506fa4 ("rename avatar_mes by avatar_message", 2022-02-21)